index.html

```diff
@@ -1031,9 +1031,15 @@ <h1 class="text-green_dark text-lg leading-snug">Pontos por noite</h1>
       </article>
 
 
-
+  
     </section>
-
+    <a href="/dashboard/buscador/"
+    style="font-weight: 400; color: #fff; background-color: rgba(69, 95, 93); border: none;box-shadow: 0px 3px 3px rgba(0, 0, 0, 0.160784);"
+      class="self-center mb-4 explorar-button button_default">
+    <p class='mt-1'>
+      Veja mais casas
+    </p>
```

Comment:
Não faz mais sentido colocar display block no <a /> e depois colocar a classe do </p> nele?
```suggestion
     <a
       href="/dashboard/buscador/"
       style="display: block; margin-top: 0.25rem, font-weight: 400; color: #fff; background-color: rgba(69, 95, 93); border: none;box-shadow: 0px 3px 3px rgba(0, 0, 0, 0.160784);"
       class="self-center mb-4 explorar-button button_default"
    >
      Veja mais casas
    </a>
 
```